### PR TITLE
Make meteor game mode great again

### PR DESCRIPTION
**Pull request by optimumtact**
Reverts tgstation/-tg-station#13907

Thanks to the new lag removal, we can make meteors great again.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
rip lag removal

**Comment by optimumtact** [comment]
Maybe we outta test merge it first then

**Comment by Zilenan91** [comment]
set it to 9 this couldn't possibly go wrong

**Comment by GunHog** [comment]
I like it. Make sure you **TEST** this locally, first.

**Comment by optimumtact** [comment]
There's no point testing this locally because I can't stress a box anywhere near what a proper test on Sybil/Basil would do.

**Comment by Dorsisdwarf** [comment]
Test-merge would work for live testing

**Comment by optimumtact** [comment]
hg test merged this about 8 hour ago

**Comment by MrStonedOne** [comment]
Make it ramp up to 6 over the course of X amount of time. like over 30 minutes.

**Comment by optimumtact** [comment]
:no_good: :rotating_light: :lock:

**Comment by Dorsisdwarf** [comment]
PR no good,  activate alarms and lock at once?

**Comment by MrStonedOne** [comment]
https://github.com/optimumtact/-tg-station/pull/6

**Comment by optimumtact** [comment]
:ticket: :green_heart: :+1:

**Comment by hornygranny** [comment]
the test merge failed when i did it actually someone test merge this

File: code/game/gamemodes/meteor/meteor.dm [hunk 1]
```diff
@@ -11,17 +12,20 @@
...
+		return
+	
+	var/list/wavetype = meteors_normal
+	var/meteorminutes = (world.time - round_start_time - meteordelay) / 10 / 60
+	
+	
+	if (prob(meteorminutes/2))
+		wavetype = meteors_catastrophic
+	else if (prob(meteorminutes))
+		wavetype = meteors_threatening
```
Comment (duncathan) [inline code/game/gamemodes/meteor/meteor.dm]: Switch the order of these two, and remove the `else`. It'll give you more accurate results as `meteorminutes` increases.

Comment (optimumtact) [inline code/game/gamemodes/meteor/meteor.dm]: https://file.house/tQG9.png

Comment (duncathan) [inline code/game/gamemodes/meteor/meteor.dm]: Yes that's what removing the else is for
The else causes that issue as well as slightly inaccurate probabilities

Comment (optimumtact) [inline code/game/gamemodes/meteor/meteor.dm]: I shall make it so then

**Comment by optimumtact** [comment]
what the fuck

**Comment by PKPenguin321** [comment]
?

**Comment by optimumtact** [comment]
Accidentally force pushed the wrong branch over this

@duncathan did what you said

I also rewrote the branch to fix up the commit history and cleaned things up a bit for readability.

**Pull request merged**